gradle.properties

```diff
@@ -79,7 +79,7 @@ ANDROIDX_LIFECYCLE_VERSION=2.4.0
 ANDROIDX_MULTIDEX_VERSION=2.0.1
 ANDROIDX_NAVIGATION_VERSION=2.3.5
 ANDROIDX_PAGING_VERSION=2.1.2
-ANDROIDX_ROOM_VERSION=2.3.0
+ANDROIDX_ROOM_VERSION=2.4.0-beta02
```

Comment:
does this update have to do with the m1 stuff?

Reply:
Yes, it has a compile-time dependency that breaks during builds.  The beta includes this fix https://issuetracker.google.com/issues/174695268